src/bounding.rs

```diff
@@ -37,9 +38,10 @@ pub fn update_bound_sphere<T: 'static + Send + Sync>(
         //Or<(Added<BoundVol>, Changed<Handle<Mesh>>)>, Broken in bevy due to unsoundness, see #9
     >,
 ) {
-    if !state.enabled {
+    if state.enabled == crate::Enabled::Disabled {
```

Comment:
See comments in lib.rs, I feel like the enum being named `Enabled` is a bit confusing in situations like this. As mentioned in that comment, the suggested change would change this into:
``` rust
if state.active == ActiveState::Disabled {
// ...
```

Reply:
Done